coalib/coala_json.py

```diff
@@ -28,12 +31,38 @@ def main():
     args = arg_parser.parse_args()
 
     log_printer = None if args.text_logs else ListLogPrinter()
-    results, exitcode, _ = run_coala(log_printer=log_printer, autoapply=False)
+    JSONEncoder = create_json_encoder(use_relpath=args.relpath)
+
+    if args.show_bears:
+        try:
+            local_bears, global_bears = get_filtered_bears(
+                args.filter_by_language, log_printer)
 
-    retval = {"results": results}
+            bears = inverse_dicts(local_bears, global_bears)
+            results = []
+            for bear, sections in sorted(bears.items(),
+                                         key=lambda bear_tuple:
+                                         bear_tuple[0].name):
+                metadata = bear.get_metadata()
+                results.append({"Name": bear.name,
+                                "Description": metadata.desc,
+                                "Supported languages": bear.LANGUAGES,
```

Comment:
@sils1297 If I append bear directly I get `TypeError: <property object at 0x7f71baf57318> is not JSON serializable`. Although `set([type(x) for x in dir(bear)])` says all the attributes are strings, so they should be serializable.